visidata/mainloop.py

```diff
@@ -295,11 +325,11 @@ def initCurses():
     colors.setup()
 
     curses.noecho()
-    curses.cbreak()
 
     curses.raw()    # get control keys instead of signals
     curses.meta(1)  # allow "8-bit chars"
-    curses.mousemask(-1 if options.mouse_interval else 0)
+    curses.MOUSE_ALL = 0xffffffff
```

Comment:
notice that we put this on the curses module for use later, so it's weird that it's not available in the other context handler.

Reply:
test_commands.py does not go through `initCurses`!

Reply:
Then let's just use `0xffffffff` as an inline constant in both places with the comment `# curses.MOUSE_ALL`.